### DISCUSSION TITLE
Notes widget for shift+space

### DISCUSSION THREAD

**Discussion by freezerain** [SnosMe/awakened-poe-trade#869] [Ideas]
Hi.
I was trying to remember which fossil spawns where in the delve so I figgured out I can just create a list of "Biome - node name - fossil - price" for top 5 fossils. However the current "search" widget is not really designed to be used as notes, for example there are only 1 line, character limit, on click event. 
Example: 
"Time-Lost Cavern/Frozen Hollow/Glyphic Fossil 90c" is one char from limit.

I am also using the same logic for cluster jewels, I have a list that called "12 nodes" with all cluster bases with 12 nodes that can cost something, and second list obviously called "8 nodes", both contains around 10 lines.

Maybe a better widget desinged to store notes and look up info will be usefull for some of the users even while it feels niche

**Comment by SnosMe** [comment]
I think notepad does the better job. 
Also you can create image and add it to cheat sheets widget.

credits: https://www.reddit.com/r/pathofexile/comments/bcjpoi/guidefossil_cheat_sheet_with_some_corrections_and/
![image](https://user-images.githubusercontent.com/4292308/212339025-062824bb-e058-4965-bbee-049b1fac89b6.png)

**Comment by MDLeide** [comment]
I came here to request the same thing. It would be very cool to stay in game while recording notes. I can see the possibility for some cool features based around this idea, but I think a very basic implementation of just a list of notes, which are simply strings, would create a lot of value.

Maybe later on you could add some formatting support, or whatever, but a way to quickly reference and edit information would be very impactful. Cheat sheets are great, but they are much more time consuming to edit.

**Reply by SnosMe** [reply]
**Steam** recently intoduced Notes in their overlay, so I don't even see a reason to reimplement it now.

**Reply by Berringer** [reply]
Not everyone is using steam to play tho :/
I would love to see a widget for notes, would be great to save different search regex:es etc. 
Notepad is great and all, but would be more great to see it inside awakened, really.
The files could still be .txt files, but like the image view we could import a .txt. Editable or not, would be sweet, as long as we could copy text from it.
Just a thought :)

**Comment by freezerain** [comment]
after more then a year I found onclick copypaste in search box function to be more important then ability to store plain text data for my cases. However I still feel widgets are a bit clunky:

-As I am runnning out of space for my nonsense I can appreceate minimizing widgets or made font smaller, maybe some compact mode style with less padding/margin. Or even tabs that works like windows/linux virtual desktop.

-widget alligning would tackle my OCD part of the brain. There are only one point for anchoring so after changing text widgets begins to overlap each other.

-widgets position in browser is shifted comparing to the game (I will attach screenshot from web, but in the game all widgets are alligned)

I am leaving my feedback on what am I personally missing, however I want to be clear there are no way I expect you to implement this atomic features that probably will require massive UI backend changes that will introduce new points of failure, and only 6 person in the world will benefit from.
But maybe some insides may affect your future decisions even not related to widgets. As I found you a little bit strict on how exactly your app should be used in the chase of purity of the codebase. 

My ridiculous overlay for reference: 

![image](https://github.com/user-attachments/assets/2a83a3f5-5e71-4cb1-8b53-4d1f3039dd14)
